config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```
@@ -1083,3 +1083,26 @@ periodics:
     testgrid-dashboards: sig-release-releng-blocking
     testgrid-alert-email: k8s-infra-alerts@kubernetes.io, release-managers@kubernetes.io
     testgrid-num-failures-to-alert: '1'
+- interval: 24h
+  cluster: test-infra-trusted
+  max_concurrency: 1
+  name: k8sio-vuln-dashboard-cron
```

```suggestion
  name: ci-k8sio-vuln-dashboard-update
```

For periodic jobs, we usually prepend `ci-`.
Add `-update` as a suffix as well, so it's a little clearer what this job is doing at-a-glance.